### DISCUSSION TITLE
Archiving of Tasks: Remand to Related Archival File?

### DISCUSSION THREAD

**Discussion by qalisto** [ransome1/sleek#309] [Q&A] [answered]
Is it possible for an option for Sleek to transfer complete tasks to a **sister** file that has a    <Tasklist>.done   extension or <Tasklist>.done.txt  ?

I believe that the Ben Hughes app did this.  Might be nice for this option (tho' I see the Utility in retaining searchability in current setup).

**Comment by ransome1** [comment, marked as answer]
Hi @qalisto sleek has an archiving function, which moves completed todos to a sister file. Just press the `A` key or click on the archiving icon displayed in every completed todo:
![image](https://user-images.githubusercontent.com/11188741/146878228-8e35d853-dc0c-4700-9f54-06c1a795ec40.png)

The sister file will be created in the same directory where you're todo file is located at. If your filename is `todo.txt` the sister file will be named `todo_done.txt`. Hope this helps.

**Reply by ransome1** [reply]
@qalisto I think this might be for your interest; The latest [developer preview of the upcoming 2.0](https://github.com/ransome1/sleek/releases/tag/v2.0.0-dev12) contains a function, which helps you defining a custom location for the done file. Feel free to join the corresponding discussion: https://github.com/ransome1/sleek/issues/457

The keyboard shortcut changed to `Ctrl + Alt + A` to avoid confusion. I also removed the icon from the table rows, since it stirred up some confusion, that had been reported. I'm still looking for a better position for the archiving function.

**Comment by qalisto** [comment]
Thank you for direction.
